src/main/scala/edu/cmu/cs/obsidian/parser/ImportProcessor.scala

```diff
@@ -12,11 +13,37 @@ object ImportProcessor {
 
     class ImportException (val message : String) extends Exception {}
 
+    def compilerPath(): Path =
+        List("TRAVIS_BUILD_DIR","OBSIDIAN_COMPILER_DIR")
```

Comment:
We should document all these environment variable names somewhere, but I realize there's aren't new in your change.

Reply:
Is there some existing place I should do this, or do we just want to create an issue for this or something?